### DISCUSSION TITLE
Exception: unknow dataset /home/zhao/pysot/testing_dataset/VOT2018/

### DISCUSSION THREAD

**Discussion by shxzhao** [STVIR/pysot#380] [General]
I download the VOT2018 dataset and put it in the testing_dataset, and get the error, how to solve it?  I appreciate it if you answer.

**Comment by ZhiyuanChen** [comment]
Could you please provide detailed error message?

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing this issue for now.
Please create a new issue should you have other questions.
